demo/lib/demo_web/live/short_link_live.ex

```diff
@@ -47,7 +47,6 @@ defmodule DemoWeb.ShortLinkLive do
       product: %{
         module: Backpex.Fields.BelongsTo,
         label: "Product",
-        source: Demo.Product,
```

Comment:
@Flo0807 This is not used anymore, right?

Reply:
yes